Bottom: 46a41435ac366ad36c28dcab41598bb149a98d57
Top:    a49d0e144acdded02c6f0d1bcaf6b39d0f7c3965
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-21 19:11:54 -0700

Refresh of rs-opt-qpsize

---

diff --git a/src/rsocket.c b/src/rsocket.c
index a03cb43..527c0dc 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -373,7 +373,6 @@ static int rs_create_ep(struct rsocket *rs)
 	qp_attr.cap.max_inline_data = RS_INLINE;
 
 	ret = rdma_create_qp(rs->cm_id, NULL, &qp_attr);
-printf("create qp %d sq %d rq %d\n", ret, rs->sq_size, rs->rq_size);
 	if (ret)
 		return ret;
